src/NzbDrone.Core/Localization/Core/en.json

```diff
@@ -215,6 +235,7 @@
   "NoLogFiles": "No log files",
   "NoMinimumForAnyRuntime": "No minimum for any runtime",
   "NoTagsHaveBeenAddedYet": "No tags have been added yet",
+  "NotifcationTriggersHelpText": "Select which events should trigger this notification",
```

Comment:
Want to spell this one right and fix it here as well https://github.com/Prowlarr/Prowlarr/blob/8b0a8e82b5973109443d5d93b657905f3963cc23/frontend/src/Settings/Notifications/Notifications/NotificationEventItems.js#L28

Then it will be in the right place in the alphabet list :D